aws-codeguruprofiler-profilinggroup/src/main/java/software/amazon/codeguruprofiler/profilinggroup/CreateHandler.java

```diff
@@ -60,6 +61,18 @@
                 ResourceModel.TYPE_NAME, pgName, awsAccountId));
         }
 
+        Optional<AnomalyDetectionNotificationConfiguration> anomalyDetectionNotificationConfiguration = anomalyDetectionNotificationConfiguration(model);
+        anomalyDetectionNotificationConfiguration.ifPresent(notificationConfiguration -> safelyInvokeApi(() -> {
+            try {
+                addChannelNotifications(pgName, notificationConfiguration.getChannels(), proxy, profilerClient);
+            } catch (CodeGuruProfilerException addChannelNotificationException) {
+                logger.log(format("%s [%s] for accountId [%s] has failed when adding Channel Notification, trying to delete the profiling group!",
+                        ResourceModel.TYPE_NAME, pgName, awsAccountId));
+                deleteProfilingGroup(proxy, logger, pgName, awsAccountId, addChannelNotificationException);
+                throw addChannelNotificationException;
+            }
+        }));
```

Comment:
Since this part of the code is quite similar to what we did with `putAgentPermissions` above, can you make sure we use the same style for naming methods and doing the operation for both?